lib/nifty_services/base_crud_service.rb

```diff
@@ -12,22 +11,25 @@ def record_type(record_type, options = {})
           record_type
         end
 
-        record_alias = options.delete(:alias_name) || record_type.to_s.underscore
+        record_alias = options.delete(:alias_name)
+        record_alias ||= record_type.to_s.underscore
 
         alias_method record_alias.to_sym, :record
       end
 
       def include_concern(namespace, concern_type)
-        _module = "#{services_concern_namespace}::#{namespace.to_s.camelize}::#{concern_type.to_s.camelize}"
+        module_name = "#{services_concern_namespace}::
```

Comment:
@viniciuszago this is better, but I'm wondering about '\n' that wil be appended to strings, see:

```
include_concern("Services::V1", "Concerns", :create)
# > Services::V1::
#                 Concerns::
#                Create

# Services::V1::\nConcerns::\nCreate
```

So this is not a valid module name